functions/php-s3/function/handler.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+require __DIR__ . "/vendor/autoload.php";
+
+function run($event, $context)
+{
+    // Get values from function environment variables
+    $s3_access_key = getenv('S3_ACCESS_KEY');
+    $s3_secret = getenv('S3_SECRET_KEY');
+    $s3_endpoint = getenv('S3_ENDPOINT');
+    $s3_region = getenv('S3_REGION');
+    $s3_bucket = getenv('S3_BUCKET');
+
+    $s3_key = 'example-key';
+
+    // Connect to S3
+    $s3 = new Aws\S3\S3Client([
```

Comment:
This is just for style but usually in PHP at the top of the file before the `require` statement we add packages, example: `use Aws\S3\S3Client;` it's useful to know all packages used in a file

Then instead of `$s3 = new Aws\S3\S3Client([` you can just write `$s3 = new S3Client([`.